### PR TITLE
Add initial implementation

### LINKED ISSUE #1: Add initial implementation

### PR DESCRIPTION
**Pull request by lhorie**
Resolves #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# fusion-cli
+
+# Installation
```
Comment (nadiia) [inline README.md]: should be level 3 header

File: README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+```sh
+yarn add fusion-cli
+```
+
+---
+
+# CLI API
```
Comment (nadiia) [inline README.md]: should be level 3 header

**Comment by AlexMSmithCA** [comment]
That's a lot of _TODOs_ #fixitWeek2017

**Comment by lhorie** [comment]
I don't think we need TODOs for adding READMEs to babel plugins, they aren't used externally by anything other than the fusionjs compiler. We do need a major refactor of this entire package, but that's tracked by #15

**Comment by lhorie** [comment]
!merge

**Pull request merged**